tests/plugins/tableselection/integrations/clipboard/xss.js

```diff
@@ -0,0 +1,44 @@
+/* bender-tags: editor,unit */
+/* bender-ckeditor-plugins: tableselection, image2 */
```

Comment:
Rather than relying on `image2` (which will wrap, and potentially modify img listeners), I think it make sense to use it without image2 plugin but adding a config:

			config: {
				allowedContent: 'img[*]'
			}

To each editor instance.